Makefile

```diff
@@ -84,11 +88,11 @@ test_unit: test_migrate test_preconditions
 
 # Integration tests
 .PHONY: test_integration
-test_integration: ## Run integration tests
```

Comment:
Why are we dropping these? Command explanations are useful.

Reply:
The comments were removed from the template. We will need to address this issue there.